.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/node:4-browsers
```

Comment:
Node.js 4 has been dropped and 6 is in maintenance since April. We should consider upgrading to 8 (current LTS).

[Node.js Release schedule](https://github.com/nodejs/Release#release-schedule)

Reply:
Yes absolutely! I wanted to avoiding changing too much in this PR, but I think that's a great point. I'll make a ticket for this in the backlog.